internal/provider/service_data_source.go

```diff
@@ -198,6 +206,13 @@ func serviceToDataModel(s *tsClient.Service) ServiceDataSourceModel {
 		},
 		Created: types.StringValue(s.Created),
 	}
+	if s.VpcEndpoint != nil {
+		if vpcId, err := strconv.ParseInt(s.VpcEndpoint.VpcId, 10, 64); err == nil {
```

Comment:
Let's handle this error somehow. Maybe just log it?

Reply:
yeah I did something dirty but no real other choice I fear